src/Core/Tools/Enums/ReferenceEventType.cs

```diff
@@ -42,6 +42,8 @@ public enum ReferenceEventType
     OrganizationEditedByAdmin,
     [EnumMember(Value = "organization-created-by-admin")]
     OrganizationCreatedByAdmin,
+    [EnumMember(Value = "organization-edited-in-stripe")]
+    OrganizationEditedInStripe,
```

Comment:
I don't see this used anywhere, should this be part of some other work? I also see it on #3305 https://github.com/bitwarden/server/pull/3305/files#diff-9b469d6a985fabbb18be0d8cc97a59addf356e1174cb0510c334c0657eb88911R45-R46

Reply:
Hi @djsmith85, yeah I just added this in anticipation that it might be used for event logging somewhere down the line, but wasn't sure how. I can remove it if that's not the case.

Reply:
An example on how to use it can be found here: https://github.com/bitwarden/server/blob/master/src/Admin/Controllers/OrganizationsController.cs#L227-L231
but if it's not needed/required please remove it, which will also remove the need for a review from @bitwarden/team-tools-dev 

As I see you changed the base-branch, you will need to remove it there

Reply:
Hi @djsmith85, apologies I might be a bit confused. I am using it in that manner in the base branch: https://github.com/bitwarden/server/blob/cc5b4ddcd4e62db344af784894976f4cba1907ee/src/Billing/Controllers/StripeController.cs#L524-L525

What I meant in my earlier comment was I didn't know if I had to wire up any additional consumption logic for the raising of the event.

Reply:
I have to apologize as I could have sworn I searched for the usage and only commented because I didn't find anything passing that newly created reference-event-type.

Nothing for me to review here anymore. Will have a look at #3305